src/test/java/NotificationMessage.java

```diff
@@ -0,0 +1,39 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.Objects;
+
+public class NotificationMessage extends BaseTest {
+    @BeforeMethod
+    public void navigate() {
+        super.setUp();
+        driver.get("http://the-internet.herokuapp.com/notification_message_rendered");
+    }
+
+    @AfterMethod
+    public void quitDriver() {
+        super.exit();
+    }
+
+    @Test
+    public void notification() {
+        boolean correctMessage = false;
+        for (int i = 0; i < 5; i++) {
```

Comment:
Why 5 iterations? not 10 or 1 or 100?)
It's OK if test will fail sometimes and pass sometimes depends on which message is displayed